drivers/adc/adc_gd32.c

```diff
@@ -174,7 +174,9 @@ static int adc_gd32_configure_sampt(const struct adc_gd32_config *cfg,
 {
 	uint8_t index = 0, offset;
 
-	if (acq_time != ADC_ACQ_TIME_DEFAULT) {
+        if (acq_time == ADC_ACQ_TIMER_MAX) {
```

Comment:
You mean `ADC_ACQ_TIME_DEFAULT`? Looks that really helpful if someone just want the biggest one.

Please help to adjust you `tab` setting. Zephyr use the similar code style like Linux,  more detail: https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style